File: 01-fizbuzz.py
while True: 
	number = int(raw_input("Enter a number: "))

	if number == -1:
		break

	if number % 3 == 0 and number % 5 == 0:
		print "FIZZBUZZ"		

	elif number % 3 == 0:
		print "FIZZ"

	elif number % 5 == 0:
		print "BUZZ"

File: 01-fizbuzz.rb
while true do
  print "Enter a number> "
  number = Integer(gets.chomp)

  if number == -1
    break
  end

  if number % 3 == 0 && number % 5 == 0
    puts "FIZZBUZZ"
  elsif number % 3 == 0
    puts "FIZZ"
  elsif number % 5 == 0
    puts "BUZZ"
  end
end
